lib/compat/wordpress-6.0/class-wp-theme-json-resolver-6-0.php

```diff
@@ -159,4 +162,153 @@ public static function get_style_variations() {
 		}
 		return $variations;
 	}
+
+	/**
+	 * Returns the custom post type that contains the user's origin config
+	 * for the current theme or a void array if none are found.
+	 *
+	 * This can also create and return a new draft custom post type.
+	 *
+	 * @param WP_Theme $theme              The theme object.  If empty, it
+	 *                                     defaults to the current theme.
+	 * @param bool     $create_post        Optional. Whether a new custom post
+	 *                                     type should be created if none are
+	 *                                     found.  False by default.
+	 * @param array    $post_status_filter Filter Optional. custom post type by
+	 *                                     post status.  ['publish'] by default,
+	 *                                     so it only fetches published posts.
+	 * @return array Custom Post Type for the user's origin config.
+	 */
+	public static function get_user_data_from_wp_global_styles( $theme, $create_post = false, $post_status_filter = array( 'publish' ) ) {
```

Comment:
Why you added these functions?

Reply:
Because these functions use the "inherited" WP_Theme_JSON_Gutenberg class. So without them it could result in the class from 5.9 being used without the overrides we have for 6.0...